atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/CheckAndSetCompatibility.java

```diff
@@ -86,6 +94,11 @@ public boolean supportsCheckAndSetOperations() {
             return false;
         }
 
+        @Override
+        public boolean supportsMultiCheckAndSetOperations() {
+            return false;
+        }
+
         @Override
```

Comment:
Should  `supportsMultiCheckAndSetOperations` be a subset of `supportsCheckAndSetOperations` i.e. if MCAS is supported, then CAS is definitely supported?

Reply:
By definition should be since you can always just do a single one